js/ui/bind_handlers.js

```diff
@@ -13,15 +13,15 @@ var handlers = {
     touchZoomRotate: require('./handler/touch_zoom_rotate')
 };
 
-module.exports = function bindHandlers(map, options) {
+module.exports = function bindHandlers(map, bindOptions, mapOptions) {
```

Comment:
Why the second `mapOptions` object? Could the `bearingSnap` property go on the original `options`/`bindOptions` object?